docs/whatsnew/4.3.rst

```diff
@@ -38,6 +38,18 @@ made much more precise (down to the milli-arcsec level).  To help this, the
 Furthermore, by ensuring expensive calculations of the precession-nutation
 matrix are only done once, the transformations have been sped up considerably.
 
+.. _whatsnew-4.3-threadsafe:
+
+Improvements in making Astropy threadsafe
+=========================================
+
+To help use of Astropy in software for the Square Kilometer Array, an effort
+has started to make Astropy threadsafe for operation that do not explicitly
+change state. In particular, parsing of units and angles, and updating
```

Comment:
```suggestion
change state. In particular, parsing of units and angles, and initializing
```
Since it's the automatic initialization that's a problem, rather than any explicit update made by the user (not sure if you prefer US or UK spelling).

Reply:
> prefer US or UK spelling

⚔️ Came up on Slack, I don't think we agreed on anything about this... ⚔️

Reply:
Being in Canada I never know which one to use...